app/views/hosts/_templates.html.erb

```diff
@@ -6,27 +6,29 @@
     </tr>
   </thead>
   <tbody>
-    <% @templates.each do |template| %>
+    <% @templates.each do |template|
+       can_preview = authorized_for(:permission => :view_provisioning_templates, :auth_object => template, :authorizer => authorizer)
```

Comment:
I guess, I should use the new  'host_provisioning_template' or 'view_host_provisioning_templates' instead of view_provisioning_templaes @MariaAga ?